numba_cuda/numba/cuda/cudadrv/driver.py

```diff
@@ -2901,7 +2922,143 @@ def complete(self):
         """
 
 
-class MVCLinker(Linker):
+class _Linker(_LinkerBase):
+    def __init__(
+        self,
+        max_registers=None,
+        lineinfo=False,
+        cc=None,
+        lto=None,
+        additional_flags=None,
+    ):
+        arch = f"sm_{cc[0] * 10 + cc[1]}"
```

Comment:
Here we do `major * 10 + minor` but elsewhere in `nvrtc.py` we are just using string concatenation like `arch = f"sm_{major}{minor}"`. I think I prefer string concatenation, but being consistent is more important.

```suggestion
        arch = f"sm_{cc[0]}{cc[1]}"
```